json-body-transformer/src/test/java/com/ninecookies/wiremock/extensions/JsonBodyTransformerTest.java

```diff
@@ -309,11 +333,21 @@ public void doesNotApplyGlobally() throws Exception {
 		wireMockServer.verify(postRequestedFor(urlEqualTo(REQUEST_URL)));
 	}
 
-	@Test
-	public void injectRandom() {
+	@DataProvider
+	private Object[][] randomFormats() {
+		return new Object[][] {
+				{ "$(!Random)" },
+				{ "$(!Random.1)" },
+				{ "$(!Random.abc)" },
+				{ "$(!Random.ABC)" },
+		};
+	}
+
+	@Test(dataProvider = "randomFormats")
+	public void injectRandom(String format) {
 		wireMockServer.stubFor(post(urlEqualTo(REQUEST_URL))
 				.willReturn(aResponse().withStatus(200).withHeader("content-type", CONTENT_TYPE)
-						.withBody("{\"randomNumber\":\"$(!Random)\", \"got\":\"it\"}")
+						.withBody("{\"randomNumber\":\"" + format + "\", \"got\":\"it\"}")
```

Comment:
Mkay. So you do some magic internally and keep it a UUID - not just appending something to it which could also work since appending 1 and 2 to the same UUID gives different values :). Not _very_ different ones, but different :P. But yeah, it wouldn't be a real UUID anymore. Same goes for Timestamps and stuff (would be suddenly strings when appending .bla).

Ok, I get that.